### ISSUE TITLE
the training codes for WCDAS+LDMLR

### ISSUE THREAD

**Issue by stylebased** [AlvinHan123/LDMLR#1]
Hi,
I am really interested in your state-of-art model WCDAS+LDMLR. Will you release codes for WCDAS+LDMLR?

**Comment by AlvinHan123** [comment]
Hi,
Many thanks for your interest! I will release WCDAS+LDMLR code as soon as possible.

**Comment by stylebased** [comment]
Thank you for your quick reply. I am also curious with the training settings for Imagenet LT. Can you also release these codes?

**Comment by stylebased** [comment]
For your Imagenet-lt dataset, I used the pretrained encoder you provided and le 1e-3 with adam in the second stage (the ldm training). However the loss is too large. Can you provide the detailed training strategy for imagenet-lt?

**Comment by AlvinHan123** [comment]
Hi please try sgd opt again.
